Vindicta.Altis/Undercover/undercoverMonitor.sqf

```diff
@@ -577,10 +580,16 @@ CLASS("UndercoverMonitor", "MessageReceiver");
 						}; // do once when state changed
 
 						if (activeACE) then { 
-							if !(_unit getVariable ["ACE_isUnconscious", false]) then { T_CALLM("setState", [sUNDERCOVER]); };
+							if !(_unit getVariable ["ACE_isUnconscious", false]) then { 
+								T_CALLM("setState", [sUNDERCOVER]); 
+								OOP_INFO_0("ACE_isUnconscious is false, leaving state sINCAPACITATED");
+							};
 						};
 
-						if (lifeState _unit == "HEALTHY") then { T_CALLM("setState", [sUNDERCOVER]); };
```

Comment:
fixed problem where UM state continuously switches between overt and undercover while unconscious